src/components/FilterSelect/index.tsx

```diff
@@ -181,6 +173,7 @@ class FilterSelect extends React.Component<Props, States> {
           aria-haspopup="true"
           isactive={value}
           onClick={this.handleClick}
+          iscapital={placeholder !== 'Select Operator'? 1 : 0}
```

Comment:
What's `iscapital` here? Seems odd to compare to a constant. That constant also should be named, not in-line.

Reply:
If this is to format the operator values, that should be done elsewhere, like a function that returns operators for a specific datatype.

Reply:
![capital](https://user-images.githubusercontent.com/27675170/56873712-69dbc480-6a02-11e9-95ef-7770a384aee5.png)

Reply:
We used the same component for the two selectors.
When it is for the operator, it should display a small character.

Reply:
`ConseilOperator` `enum` in ConseilJS has these in lower case, but you'd still need to format "startWith" into "starts with". Maybe implement a utility function for this that takes a data type as parameter and returns the display values or display tuples like `[{'startsWIth': 'starts with'}, 'in', 'in']`, etc.